src/ontology/edits/oeo-physical.omn

```diff
@@ -426,6 +630,43 @@ Class: <http://openenergy-platform.org/ontology/oeo-physical/RenewableFuel>
         <http://purl.obolibrary.org/obo/RO_0000091> some CombustibleEnergyCarrierDisposition
     
     
+Class: <http://openenergy-platform.org/ontology/oeo-physical/SolarPowerUnit>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A SolarPowerUnit is a RegenerativePowerUnit using solar power."
+    
+    SubClassOf: 
+        <http://openenergy-platform.org/ontology/oeo-physical/RegenerativePowerUnit>
+    
+    
+Class: <http://openenergy-platform.org/ontology/oeo-physical/SolarThermalPowerUnit>
+
+    Annotations: 
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A SolarThermalPowerUnit is a SolarPowerUnit using solar thermal energy."
```

Comment:
I think we need here a different definition. Something like:
_A SolarThermalPowerUnit is a SolarPowerUnit that has part SolarThermalCollectors._

Reply:
ok I implemented it as: "A SolarThermalPowerUnit is a SolarPowerUnit that has SolarThermalCollectors as parts." (reads a bit easier) and added the has_part relation